lib/ui/common/ActionSheet.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:farmsmart_flutter/ui/common/ActionSheetLargeRoundedButtonStyle.dart';
+import 'package:farmsmart_flutter/ui/common/ActionSheetListItem.dart';
+import 'package:farmsmart_flutter/ui/common/ListDivider.dart';
+import 'package:farmsmart_flutter/ui/common/headerAndFooterListView.dart';
+import 'package:farmsmart_flutter/ui/common/roundedButton.dart';
+import 'package:farmsmart_flutter/ui/mockData/MockActionSheetViewModel.dart';
+import 'package:flutter/material.dart';
+
+class ActionSheetViewModel {
+  List<ActionSheetListItemViewModel> actions;
+  String confirmButtonTitle;
+  String cancelButtonTitle;
+
+  ActionSheetViewModel(this.actions, this.confirmButtonTitle,
+      {this.cancelButtonTitle});
+}
+
+class ActionSheetStyle {
+  final Color cornersColor = const Color(0xFF737373);
+  final Color backgroundColor;
+  final Color indicatorLineColor;
+  final Color confirmButtonBackgroundColor;
+  final Color cancelButtonBackgroundColor;
+
+  final EdgeInsets indicatorLineEdgePadding;
+
+  final Radius cornerRadius;
+  final Radius indicatorLineRadius;
+
+  final double indicatorLineThickness;
+  final double indicatorLineWidth;
```

Comment:
width, and the edge padding above? do we need both?

Reply:
Yes, it's needed because, edge padding is now implemented as margin, so is setting the top and bottom margin for the indicator